### PR TITLE
Show Copied info when an identicon is clicked

### LINKED ISSUE #92: Show "Copied" info when an identicon is clicked
Input from users:

I don't understand why the cursor of the mouse has a "+" when it's hovering an identicon.
This is a behavior from the @polkadot/react-identicon repo. If clicked, the address of the account will be copied to the clipboard. 
Users need to understand this.

Proposal: show a "copied" tool-tip above or below the identicon when clicked. It should be visible for ~5 sec
Alternatively we could use a toast with the toast context `useToasts`

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
closes #92

Fixes: i have added all the required component but don't know why it's not working.

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -18,6 +19,7 @@
     "react-dom": "18.2.0",
     "react-query-subscription": "2.0.0-beta.1",
     "react-router-dom": "^6.10.0",
+    "react-tooltip": "^5.11.1",
```
Comment (Tbaut) [inline ui/package.json:22]: you do not need to add either matrial-iu/core, nor react-tooltip.
Please remove them

File: ui/src/components/AccountSelect/index.tsx [hunk 2]
```diff
@@ -36,11 +37,9 @@ export const AccountSelect: React.FC<Props> = ({ anchorEl, onClose, ...props })
           <CardHeader
             avatar={
               <IconButton sx={{ p: 0 }}>
-                <Identicon
-                  value={account.address}
-                  theme="substrate"
-                  size={40}
-                />
+                   
+                  <MultixIdenticon value={account.address} theme="substrate" size={40} className={""} />
```
Comment (Tbaut) [inline ui/src/components/AccountSelect/index.tsx:41]: On several lines please, as it was for `Identicon`

File: ui/src/components/MultixIdenticon.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import Tooltip from "@material-ui/core/Tooltip";
+import Identicon from "@polkadot/react-identicon";
+import { AccountSelect } from "./AccountSelect";
+
+
+export default function MultixIdenticon({value,theme,size,className}) {
```
Comment (Tbaut) [inline ui/src/components/MultixIdenticon.tsx:7]: This is a typescript file, you must have types for the props.

File: ui/src/components/MultixIdenticon.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <>
```
Comment (Tbaut) [inline ui/src/components/MultixIdenticon.tsx:9]: No need for this, the Tooltip only one component.

**Review by Tbaut — CHANGES_REQUESTED** [review]
From my point of view, you are close to have something working.
Please make sure to type your props, this is TS, not JS.
also in terms of style, please have the `MultixIdenticon` props on a separate line for each (I left one comment but this is for all occurences).

File: ui/package.json [hunk 5]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.12.4",
```
Comment (Tbaut) [inline ui/package.json:6]: This is not needed, you should use
```
import Tooltip from '@mui/material/Tooltip';
```

File: ui/src/components/MultixIdenticon.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        title="Copied"
+        placement="top"
+        arrow
+      >
+      <Identicon
+               value={value}
+               theme={theme}
+               size={size}
+               className={className}
+            ></Identicon>
```
Comment (Tbaut) [inline ui/src/components/MultixIdenticon.tsx:25]:
```suggestion
            />
```

File: ui/src/components/MultixIdenticon.tsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+interface Props {
+  value?: string,
+  theme?: any,
+  size: number,
+  className?: string;
+}
+const MultixIdenticon = ({value,theme,size,className} : Props) => {
+  
+  return (
+    <div>
```
Comment (Tbaut) [inline ui/src/components/MultixIdenticon.tsx:14]: this `div` wrapper is not needed as far as I can tell, please remove.

**Review by Tbaut — CHANGES_REQUESTED** [review]
Please make sure that:
- MultixIdenticon props are on new lines
- run `yarn lint` or `yarn lint --fix` and fix any error

File: ui/src/components/MultixIdenticon.tsx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+      <Tooltip
```
Comment (Tbaut) [inline ui/src/components/MultixIdenticon.tsx:15]: This adds a constant tooltip to the icon eventually. There is no notion of "appearing on click", or being invisible after a couple seconds. You need to take this into consideration.

**Comment by tarunsamanta2k20** [comment]
@Tbaut no onClick() found  for yarn add @polkadot/react-identicon.

**Comment by tarunsamanta2k20** [comment]
@Tbaut somehow i did. But for that I had to move the icon button along with Identicon. watch my demo once. 

https://user-images.githubusercontent.com/55488549/234203837-8ee0924d-7a59-41dc-aff1-099d144aafc9.mp4

**Comment by Tbaut** [comment]
The video doesn't display for me. You need to wrap the identicon and catch the click there.

**Comment by tarunsamanta2k20** [comment]
> The video doesn't display for me. You need to wrap the identicon and catch the click there.

Please watch it on laptop or desktop

**Comment by Tbaut** [comment]
The file is corrupt, the video can't be seen on anything for me.
image

**Comment by tarunsamanta2k20** [comment]
@Tbautp Check my latest commit once

**Comment by tarunsamanta2k20** [comment]
> The file is corrupt, the video can't be seen on anything for me. image

@Tbaut i am watching this video without any issue.

**Comment by Tbaut** [comment]
This type of Ai assisted, low quality submissions will not be dealt with in the future. Thanks for the contributions.

**Pull request closed without merging**